pyspeckit/spectrum/classes.py

```diff
@@ -181,14 +180,12 @@ def __init__(self, filename=None, filetype=None, xarr=None, data=None,
                 self.data = np.ma.masked_where(np.isnan(self.data) + np.isinf(self.data), self.data)
                 self.error = np.ma.masked_where(np.isnan(self.data) + np.isinf(self.data), self.error)
 
-        # it is very important that this be done BEFORE the spectofit is set!
-        self._sort()
-
```

Comment:
I see, the merge errors worry me a bit, I'll try to find the commit in your branch where it happened and make sure I didn't skip anything. I believe the error was caused by me using git checkout --ours and --theirs